erpnext_documentation/www/docs/user/manual/en/human-resources/setting-up-tax.md

```diff
@@ -1,34 +1,34 @@
+# Setting Up Income Tax Deduction
+Calculating Tax deductions for employees every month is a time-consuming activity for most businesses, especially for large enterprises. If set up properly, ERPNext simplifies most of the tax-related calculations by automatically calculating tax deductions while generating Salary Slips. Here's how you can configure ERPNext to ease your payroll processing -
 
-#Income Tax Exemption
-In many countries, especially in India, regulations allow exempting a part (or all) of some type of spendings by individuals from being added to their annual taxable income. Examples of such spendings could be contributions to charitable institutions, amount spent for education of children, specific investments etc. To
+# Income Tax Exemption
+In many countries, especially in India, regulations allow exempting a part (or all) of some type of spendings by individuals from being added to their annual taxable income. Examples of such spendings could be contributions to charitable institutions, the amount spent on the education of children, specific investments, etc. To
 avail the exemption from their taxable income, individuals are required to submit proof of such spendings.
 
-ERPNext allows you to configure Tax Slabs as part of every Payroll Period and the applies tax on each Salary based on projected annual earnings of the employee. Tax thus calculated will be added to the Salary Slip for  deduction Salary Component configured _Variable Based On Taxable Salary_. Employees are required to declare the exemption amount they plan to claim at the end of the fiscal year so that the payroll deductions for tax will be calculated based on the projected annual earnings less the exemption. Employees can declare this through [Employee Tax Exemption Declaration](/docs/user/manual/en/human-resources/employee-tax-exemption-declaration), the total of which will be exempted from annual taxable earnings of the employee.
+ERPNext allows you to configure Income Tax Slabs and the applies to the tax on each Salary based on projected annual earnings of the employee. The tax thus calculated will be added to the Salary Slip for deduction Salary Component configured _Variable Based On Taxable Salary_. Employees are required to declare the exemption amount they plan to claim at the end of the fiscal year so that the payroll deductions for tax will be calculated based on the projected annual earnings less than the exemption. Employees can declare this through [Employee Tax Exemption Declaration](/docs/user/manual/en/human-resources/employee-tax-exemption-declaration), the total of which will be exempted from annual taxable earnings of the employee.
```

Comment:
ERPNext allows you to configure Income Tax Slabs and the tax is calculated based on the projected annual earnings of the employee. For this, employees are required to declare the exemption amount they plan to claim at the start of the financial year so that the payroll deductions for tax will be calculated based on the projected annual earnings minus the exemption.